lib/api/v3/versions/versions_api.rb

```diff
@@ -36,7 +36,7 @@ class VersionsAPI < ::API::OpenProjectAPI
                                                             # the distinct(false) is added in order to allow ORDER BY LOWER(name)
                                                             # which would otherwise be invalid in postgresql
                                                             # SELECT DISTINCT, ORDER BY expressions must appear in select list
-                                                            Version.visible(current_user).distinct(false)
+                                                            Version.visible(current_user).or(Version.systemwide).distinct(false)
```

Comment:
In the long run it would be better to have Version.visible include the systemwide versions to avoid problems like this in the future.